app/src/ui/dropdown-select-button.tsx

```diff
@@ -121,25 +140,22 @@ export class DropdownSelectButton<
       return
     }
 
-    const foundCurrentIndex = [...buttons].findIndex(b =>
-      b.className.includes('focus')
-    )
+    const foundCurrentIndex =
+      document.activeElement === null
+        ? -1
+        : [...buttons].indexOf(document.activeElement)
```

Comment:
Not sure if using `document.activeElement` is the best way, but for some reason the class `focus` is not applied anymore. I couldn't find if that was a Chrome thing (and therefore an Electron upgrade broke it) or if it was something custom we did.

In any case, using `document.activeElement` works and we also do a similar check [in the TextBox component](https://github.com/desktop/desktop/blob/4fd78b52bbca68aa291899329d64b4164754491d/app/src/ui/lib/text-box.tsx#L146)

Reply:
Yeah.. I feel like I vaguely remember a few months back something about focus stuff changing. Maybe this was impacted then too.